Dockerfile

```diff
@@ -1,13 +0,0 @@
-FROM tozd/runit
```

Comment:
Please just add it the file, keep the old one around (or rename old one to `mongodb-2.6.dockerfile`).